src/static/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+	"short_name": "Open Collective",
+	"name": "Open Collective",
+	"icons": [
+		{
+			"src": "https://opencollective.com/favicon.icon",
+			"sizes": "192x192",
+			"type": "image/png"
+		}
+	],
+	"start_url": "/",
+	"display": "standalone",
+	"theme_color": "#ffffff",
+	"background_color": "#ffffff"
+}
```

Comment:
Small thing, this is not formatted by prettier.

Can you add `!src/static/*.json` at the end of `.prettierignore` and run `npm run prettier:write`

Reply:
done